### ISSUE TITLE
What is the difference between Vc_IMPL and TARGET_ARCHITECTURE?

### ISSUE THREAD

**Issue by adriansev** [VcDevel/Vc#189]
vc 1.3.3
I am trying to compile a vc that would have the lowest denominator from a bunch of machines (that would be a TARGET_ARCHITECTURE=core) 
But it seems that no setting the Vc_IMPL will set it as auto, so in opposition with TARGET_ARCHITECTURE .. which one should be set? 
Thank you!
Adrian

**Comment by mattkretz** [comment]
Hi, sorry for the late answer - hopefully not too late.
In general, when you compile Vc itself, the setting only matters for building the unit tests and examples. When you build an application using Vc you should just make sure you use the `-march` flag that best matches your targets. You can do that without the Vc supplied cmake scripts for full control.
I don't understand "it seems that no setting the Vc_IMPL will set it as auto". Can you rephrase the question?
